besu/src/main/java/org/hyperledger/besu/cli/util/EnvironmentVariableDefaultProvider.java

```diff
@@ -36,6 +36,10 @@ public EnvironmentVariableDefaultProvider(final Map<String, String> environment)
 
   @Override
   public String defaultValue(final ArgSpec argSpec) {
+    if (!argSpec.isOption()) {
```

Comment:
What is the reasoning for using the negation of `isOption()` instead of simply using `isPositional()` to identify a positional param?

Reply:
fair point ... I think I saw an example which was using `isOption` which stuck in my head, i didn't realised about isPositional. I'll change it to use isPositional. 👍